src/client/Microsoft.Identity.Client/TelemetryCore/OpenTelemetry/IOtelInstrumentation.cs

```diff
@@ -14,13 +14,17 @@ internal interface IOtelInstrumentation
         internal void LogSuccessMetrics(
             string platform,
             ApiEvent.ApiIds apiId,
+            string callerSdkId,
+            string callerSdkVersion,
             CacheLevel cacheLevel,
             long totalDurationInUs,
             AuthenticationResultMetadata authResultMetadata,
             ILoggerAdapter logger);
 
         internal void IncrementSuccessCounter(string platform,
             ApiEvent.ApiIds apiId,
+            string callerSdkId,
+            string callerSdkVersion,
             TokenSource tokenSource,
             CacheRefreshReason cacheRefreshReason,
             CacheLevel cacheLevel,
```

Comment:
Why not add the caller SDK/version on the log failure below as well? What if for some reason 100% of calls fail with a new version?